src/Commands/WikiBackup.php

```diff
@@ -438,6 +557,37 @@ private function outputEndInfo( $output ) {
 
 	private function removeOldBackups() {
 		$backupdirManager = new BackupDirManager( $this->dest, $this->output );
-		$backupdirManager->removeOldFiles( $this->wikiName, $this->maxBackupFiles );
+		$backupdirManager->removeOldFiles( $this->getTargetFilename(), $this->maxBackupFiles );
+	}
+
+	/**
+	 * @param array $prefixes
+	 * @param bool $includeViews
+	 * @return array
+	 */
+	private function getTablesWithPrefix( array $prefixes, bool $includeViews = false ): array {
```

Comment:
It is not clear to me when this is used and when it isn't. Looks like some Extension:DynamicPageList thingy.